app/components/account-card.js

```diff
@@ -0,0 +1,81 @@
+import Ember from 'ember';
+
+var computed = Ember.computed;
+
+export default Ember.Component.extend({
+	accountManager: Ember.inject.service('accountManager'),
+
+	purecloud: Ember.inject.service('purecloud'),
+
+	active: computed('accountManager.selectedAccount', function () {
+		return this.get('account').status;
+	}),
+
+	lastAccount: computed('accountManager.finalAccount', function () {
+		this.get('accountManager.finalAccount');
```

Comment:
Should there be a return here?

Reply:
That property is not doing anything now but deleting causes a bug I don't understand why

Reply:
`lastAccount` is being used in app/templates/components/account-card.hbs. Deleting the property here and not in the component would result in an error.

Reply:
I tried delete both and the error persisted. I will try again just to make sure

Reply:
What's the status on this? Functionally, the project works but this code still looks a little suspect without a return statement. Is the finalaccount property being used anymore?

Reply:
It is not used anymore but when removed, every account card gets selected(green color gets applied). I tried to understand but never got an insight that is why I left it.